word_guess.rb

```diff
@@ -0,0 +1,159 @@
+class WordGuess
+  attr_accessor :solve_word, :number_of_lives
+    def initialize(hash)
```

Comment:
Watch your indentation here - the `initialize` method should be in line with the attr_accessor